pkgs/tools/misc/plantuml/default.nix

```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation rec {
     homepage = "http://plantuml.sourceforge.net/";
     # "plantuml -license" says GPLv3 or later
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ bjornfor ];
+    maintainers = with maintainers; [ bjornfor Mogria ];
```

Comment:
I was really confused that the user of this commit and the author of PR doesn't match, but I will just assume this is another account.